pyproject.toml

```diff
@@ -27,15 +27,15 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pyright]
-reportMissingTypeStubs = true
+reportMissingTypeStubs = false  # Changed from true
```

Comment:
oh I didn't know there's a setting for this so I just used `#type ignore`, I'll add this to my own repo :P :+1: :+1: